lib/messages.js

```diff
@@ -827,6 +829,104 @@ util.inherits(GetHeaders, GetBlocks);
 module.exports.GetHeaders = Message.COMMANDS.getheaders = GetHeaders;
 
 /**
+ * Request peer to apply a bloom filter to 'inv' messages sent back
+ *
+ * @name P2P.Message.filterload
+ * @param{BloomFilter} filter - a BloomFilter object
+ */
+function FilterLoad(filter) {
+  this.command = 'filterload';
+  $.checkArgument(_.isUndefined(filter) || typeof filter == 'object',
+    'Object or undefined required for filter object');
+  // TODO: Are these defaults (1,0.01) sane?
+  this.filter = filter || BloomFilter.create(1, 0.01);
```

Comment:
One element is fairly unlikely, that false positive rate is though (from bitcoin core tests). This one is going to be difficult to have a good default, so we may be okay here to require it.